sos/plugins/__init__.py

```diff
@@ -180,14 +180,17 @@ def is_installed(self, package_name):
         '''Is the package $package_name installed?'''
         return self.policy().pkg_by_name(package_name) is not None
 
-    def do_cmd_output_sub(self, cmd, regexp, subst):
+    def do_cmd_output_sub(self, cmd, regexp, subst, dotall=False,
```

Comment:
This all needs to be in a separate patch - your patch description and subject are misleading because they do not document these changes in `Plugin` at all.

Split it out as a separate commit using `git rebase -i`, add a proper descriptoin, and repush the branch with `git push --force`.